Bookish.DataAccess/Services/BookishService.cs

```diff
@@ -115,6 +117,20 @@ public bool DoesIsbnExist(string isbn)
                 .Any();
         }
 
+        public bool IsCopyAvailable(int copyId)
+        {
+            var query = "SELECT LenderID FROM Loans WHERE CopyID = @CopyID";
+
+            return !dbConnection.Query<string>(query, new { CopyID = copyId }).Any();
+        }
+
+        public string GetCopyLender(int copyId)
```

Comment:
Return type should be nullable string I think, as there might not be a lender? 🙂

Reply:
Online resources say strings should already nullable, is this something I should add for readability?

Reply:
Yes, you should mark them as nullable `string?`. Then you'll get warnings if you don't check for null when using this method, which you wouldn't otherwise. It's best practice to always mark something as nullable if it can be null, and not set it to null otherwise. Does that make sense? 🙂